mu4/scores/qml/MuseScore/Scores/icons/add.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="100" height="100">
```

Comment:
@vpereverzev  Do we use font icons everywhere? We should write an article about this in the documentation

Reply:
Yes, we do

Regarding the documentation - I agree with you

Reply:
Thanks. Fixed it